.experimenter.yaml

```diff
@@ -1,228 +0,0 @@
----
-contextual-hint-feature:
```

Comment:
How come the entire '.experimenter.yaml' file gets removed?

Reply:
> the `generate-experimenter` command is now being run by experimenter, so is no longer needed to be commited into this repository.

It's only function was to be ingested by experimenter, and now experimenter can ingest the FML directly.

Reply:
Amazing!